binding/src/main/java/com/virtuslab/binding/RuntimeBinding.java

```diff
@@ -0,0 +1,38 @@
+package com.virtuslab.binding;
+
+import static lombok.Lombok.sneakyThrow;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.reflections.Reflections;
+
+public final class RuntimeBinding {
```

Comment:
DI for the poor xD

Reply:
this part of the PR is the best! :rocket: